virtio-devices/src/vhost_user/fs.rs

```diff
@@ -131,6 +131,7 @@ impl VhostUserMasterReqHandler for SlaveReqHandler {
             // of the entire mapping.
             if len == 0xffff_ffff_ffff_ffff {
                 len = self.cache_size;
+                offset = 0;
```

Comment:
Is this a bug, how did you trigger the bug, how did you check  that the bug was fixed if you had a way to reproduce the bug?

Better:

```
// Need to handle a special case where the slave ask for the unmapping
// of the entire mapping.
let offset = if len == 0xffff_ffff_ffff_ffff {
  len = self.cache_size;
    0
  } else {
    fs.cache_offset[i];
  }
```

Reply:
> Is this a bug, how did you trigger the bug, how did you check that the bug was fixed if you had a way to reproduce the bug?
> 
> Better:
> 
> ```
> // Need to handle a special case where the slave ask for the unmapping
> // of the entire mapping.
> let offset = if len == 0xffff_ffff_ffff_ffff {
>   len = self.cache_size;
>     0
>   } else {
>     fs.cache_offset[i];
>   }
> ```

Hi Rob, I didn't trigger it for real, just did this based on code analysis. The DAX part seems inactivated yet. But I did see similar logic in other hypervisor's code, which has this 'offset = 0'

Reply:
Okey dokey.